x-pack/test/apm_api_integration/common/config.ts

```diff
@@ -17,7 +17,7 @@ import { registry } from './registry';
 interface Config {
   name: APMFtrConfigName;
   license: 'basic' | 'trial';
-  kibanaConfig?: Record<string, string>;
+  kibanaConfig?: Record<string, string | string[]>;
```

Comment:
Added the above while I was debugging the integration tests using the following as part of the FTR kibana config and needed to pass in an array of values instead of just a string. I can remove this but figured it might be useful going forward for anybody who is debugging tests in the future.


`--logging.verbose=true`,
          `--logging.events.log=${JSON.stringify([
            'alerts',
            'ruleRegistry',
            'apm',
            'securitySolution',
            'info',
            'warning',
            'error',
            'fatal',
          ])}`,
          `--logging.events.request=${JSON.stringify(['info', 'warning', 'error', 'fatal'])}`,
          `--logging.events.error='*'`,
          `--logging.events.ops=__no-ops__`,